data/defikingdoms_dex/tokens.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+file = open("liquidity_pools.json")
+data = json.loads(file.read())
+
+dups = []
+
+print("token_address,token_name,token_symbol,decimals")
+for lpPool in data["data"]["pairs"]:
+    if (lpPool["token0"]["id"] in dups) == False:
```

Comment:
if not using set here the following is better here
```suggestion
    if not (lpPool["token0"]["id"] in dups):
```